.github/workflows/chromatic-pr.yml

```diff
@@ -1,13 +1,13 @@
 # Build a branch-specific Storybook instance on Chromatic
-# To trigger, add the 'chromatic' label to the pull request
-# To re-trigger, remove and add the 'chromatic' label to the pull request
+# Any file that has "__chromatic-sync" appended to the end of the filename
```

Comment:
Awesome work @etanb! I have a question and this might be a silly question since I don't know a lot about Chromatic but why is the `__chromatic-sync` appending required? I would imagine there would be a way to have Chromatic automatically sync any component that's pushed in a PR. Or maybe not? I'm mostly just curious

Reply:
I changed the "magic updating" to be if a file with the first line comment being `// AutoUpdateFileChromatic` is checked into a PR, then update Chromatic.

We can easily make Chromatic launch on EVERY PR, but the huge majority of our file changes don't involve component updates. This way, whenever specific files with the special `AutoUpdateFileChromatic` comment are updated, Chromatic is automatically triggered.